content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: kubectl Cheat Sheet
+title: kubectl Quick Guide
```

Comment:
This only fixes the title of the page, I still have to make a call to kubectl/cheatsheet endpoint though
![image](https://github.com/kubernetes/website/assets/12787482/4bc55954-1675-457c-a3f8-cafccbabf587)

Reply:
thanks for suggestion Gaurav

Reply:
can you please tell me that the issue will solve by just changing filename

Reply:
Yes, It should. Push and I'll check again for you

Reply:
can you please check it again